src/main/java/com/paneedah/mwc/network/messages/RealisticSoundClientMessage.java

```diff
@@ -0,0 +1,62 @@
+package com.paneedah.mwc.network.messages;
+
+import io.netty.buffer.ByteBuf;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import net.minecraft.util.ResourceLocation;
+import net.minecraft.util.SoundEvent;
+import net.minecraft.util.math.BlockPos;
+import net.minecraftforge.fml.common.network.simpleimpl.IMessage;
+
+
+@Getter
+@NoArgsConstructor
+@AllArgsConstructor
+public class RealisticSoundClientMessage implements IMessage {
+
+    private SoundEvent sound;
+    private BlockPos pos;
+    private float volume;
+    private float pitch;
+    private double distance;
```

Comment:
Weird obsession with ordering 
```suggestion
    private float volume;
    private float pitch;
    
    private double distance;
    
    private SoundEvent sound;
    private BlockPos pos;
```